src/prefect/cli/run.py

```diff
@@ -66,8 +69,22 @@ def run():
 @click.option(
     "--logs", "-l", is_flag=True, help="Live logs of the flow run.", hidden=True
 )
+@click.option(
+    "--flow-run-id-only",
```

Comment:
This is very descriptive but also long; not a blocker but what about `--no-url`?  The main benefit would be a common flag / kwarg for our CLI commands which optionally output URLs

Reply:
Hahaha I like that shorter option much better